python/graphscope/client/utils.py

```diff
@@ -83,6 +83,33 @@ def update():
 GSLogger.init()
 
 
+class CaptureKeyboardInterrupt(object):
+    """Context Manager for capture keyboard interrupt
+
+    Args:
+        func: function
+            Callback function when KeyboardInterrupt occurs.
+
+    Examples:
+        >>> with CaptureKeyboardInterrupt(func):
+        >>>     do_somethings()
+    """
+
+    def __init__(self, func=None):
+        self._func = func
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_value, exc_tb):
+        if exc_type is KeyboardInterrupt:
```

Comment:
Otherwise we don't need such cleanup?

I remember that if the `connect` throws exceptions the kubernetes namespace won't be correctly (though I doesn't remember it clearly).

Reply:
Great suggestion. I have captured all exceptions here.